src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MetricsEventSource.cs

```diff
@@ -217,6 +225,16 @@ public void UpDownCounterRateValuePublished(string sessionId, string meterName,
             WriteEvent(16, sessionId, meterName, meterVersion ?? "", instrumentName, unit ?? "", tags, rate, value);
         }
 
+        [Event(17, Keywords = Keywords.TimeSeriesValues)]
+#if !NET8_0_OR_GREATER
```

Comment:
I'm lacking some context here since this change happened midway through my change, but it appears that builds were failing due to this not being here - let me know if this isn't the correct resolution

Reply:
This looks good, we made a change in EventSource to get rid of most of these suppression attributes, so after that change it would not be required except on below 8.